yeat/tests/test_cli.py

```diff
@@ -195,3 +196,32 @@ def test_random_downsample_seed(execution_number, capsys, tmp_path):
     assert largest_contig == pytest.approx(5228, abs=1045)  # 5228 +/- 20%
     total_len = df.iloc[14]["sample_contigs"]
     assert total_len == pytest.approx(74393, abs=14878)  # 74393 +/- 20%
+
+
+@pytest.mark.long
+@pytest.mark.parametrize(
+    "inread1,inread2",
+    [
+        ("short_reads_1.fastq", "short_reads_2.fastq"),
+        ("short_reads_1.fastq.gz", "short_reads_2.fastq"),
+    ],
+)
+def test_uncompressed_input_reads(inread1, inread2, capfd, tmp_path):
```

Comment:
Added a test that tests:

1. 2 uncompressed files
2. 1 uncompressed + 1 compressed files